<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timer</class>
 <widget class="QMainWindow" name="Timer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timer</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Hour_Label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Jokerman&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Minute_Label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Jokerman&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>350</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Hour_Line_Edit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>113</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: 20pt &quot;Jokerman&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Minute_Line_Edit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>113</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: 20pt &quot;Jokerman&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Second_Label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Jokerman&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Seconds_Line_Edit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>113</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: 20pt &quot;Jokerman&quot;;</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="Timer_bar">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>21</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>21</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Hour_error_message">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;Jokerman&quot;;
color: &quot;Red&quot;;
setVisibility:False;</string>
    </property>
    <property name="text">
     <string>Hour invalid</string>
    </property>
   </widget>
   <widget class="QLabel" name="Minute_error_message">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;Jokerman&quot;;
color: &quot;Red&quot;;</string>
    </property>
    <property name="text">
     <string>Minute Invalid</string>
    </property>
   </widget>
   <widget class="QLabel" name="Second_error_message">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;Jokerman&quot;;
color: &quot;Red&quot;;</string>
    </property>
    <property name="text">
     <string>Second Invalid</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Background_Line_Edit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Background_change_button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>Change background image</string>
    </property>
   </widget>
   <widget class="QLabel" name="Settings_label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="AppLogo_Line_Edit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AppLogo_change_button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>200</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>Change app logo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Settings_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Return_button_to_timer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Restart_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>390</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Change_Alarm_button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>Change Alarm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Change_Alarm_Line_Edit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Alarm_error_message">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;Jokerman&quot;;
color: &quot;Red&quot;;
setVisibility:False;</string>
    </property>
    <property name="text">
     <string>Alarm invalid</string>
    </property>
   </widget>
   <widget class="QLabel" name="Background_error_message">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;Jokerman&quot;;
color: &quot;Red&quot;;
setVisibility:False;</string>
    </property>
    <property name="text">
     <string>Background invalid</string>
    </property>
   </widget>
   <widget class="QLabel" name="App_Logo_error_message">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;Jokerman&quot;;
color: &quot;Red&quot;;
setVisibility:False;</string>
    </property>
    <property name="text">
     <string>App logo invalid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="To_Do_List_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>To Do list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_To_do_task_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Item_Added_to_do_line_edit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
   </widget>
   <widget class="QListWidget" name="To_Do_list_listWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>321</width>
      <height>291</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete_all_tasks_done">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>Delete All completed tasks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete_To_do_task_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>440</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>Delete task</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Item_Deleted_to_do_line_edit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="Not_Found_task_error">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;Jokerman&quot;;
color: &quot;Red&quot;;
setVisibility:False;</string>
    </property>
    <property name="text">
     <string>Not Found</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Return_to_do_to_main_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="Already_existing_task_error">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px &quot;Jokerman&quot;;
color: &quot;Red&quot;;
setVisibility:False;</string>
    </property>
    <property name="text">
     <string>Already Exists</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTimer">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menuTimer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
